.envrc.example

```diff
@@ -0,0 +1,18 @@
+# Django secret key used on app settings
+export SECRET_KEY="your_secret_here"
+
+# authentication for koiki API.
+export KOIKI_HOST="https://rekistest.koiki.es/services"
+export KOIKI_AUTH_TOKEN="your_auth_token here"
+
+# wordpress woocommerce auth
+export WCFMMP_HOST="your_wp_woocomerce_url"
+export WCFMMP_USER="your_wp_woocomerce_user"
+export WCFMMP_PASSWORD="your_wp_woocomerce_password"
+
+# if you use redis with docker as described on documentation that's your url
```

Comment:
`(...) as described on the README (...)` :ok_hand:

Reply:
Fixed by https://github.com/coopdevs/lazona_connector/pull/92/commits/944c8d64dd801e58db7de748881a9d7a554a3b51